content/learn/book/welcome/_index.md

```diff
@@ -6,4 +6,44 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: Briefly discuss what Bevy is.
+Welcome to the Bevy game engine! A bevy may be a group of birds, but our engine is:
+
+- Rust-first: with all the safety, tooling and power that comes with a modern systems language
+- completely free, open source and hackable with an architecture designed for modularity
+- powered by an ergonomic and innovative ECS that makes writing clear, performant gameplay and engine logic a joy
+- cross-platform: support all major desktop platforms with a single code-base, with limited mobile and web support
```

Comment:
"limited mobile and web support" in a list of features / design goals doesn't get the point across well enough (it reads  more like a fundamental limitation / consequence of the design). I think we should rephrase this more optimistically:

> publish Bevy Apps to Windows, MacOS, Linux, iOS*, Android*, and Web* using a single code base. Platforms with (*) are still a work in progress and are missing some features.

I also think iOS might be able to graduate to "first class" support. It has audio and asset support now and I'm not aware of any limitations.